python-spec/src/somacore/query/query.py

```diff
@@ -342,8 +399,29 @@ def _read(
             for _xname in all_x_arrays
         }
 
+        obsm = obsm_ft.result()
+        obsp = obsp_ft.result()
+        varm = varm_ft.result()
+        varp = varp_ft.result()
```

Comment:
It seems like these could be passed as the named arguments to `_AxisQueryResult` without the need for a temporary variable.